proto/kava/community/v1beta1/tx.proto

```diff
@@ -27,3 +31,17 @@ message MsgFundCommunityPool {
 
 // MsgFundCommunityPoolResponse defines the Msg/FundCommunityPool response type.
 message MsgFundCommunityPoolResponse {}
+
+// MsgUpdateParams allows an account to update the community module parameters.
+message MsgUpdateParams {
+  option (gogoproto.goproto_getters) = false;
+
+  // authority is the address that controls the module (defaults to x/gov unless overwritten).
+  string authority = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+
+  // params defines the x/distribution parameters to update.
```

Comment:
nit: Probably worth making the doc strings clearer (x/community instead of x/distribution?), as we'll likely copy and paste them when we update to the new params in other modules.